### ISSUE TITLE
atlasmap shown as the route for the syndesis app in the openshift console

### ISSUE THREAD

**Issue by gashcrumb** [syndesisio/openshift-templates#103]
Not sure if this is exactly the right repo to log this, but noticed in the openshift console the app route is wrong:

![capture](https://user-images.githubusercontent.com/351660/31021356-969dbe9e-a503-11e7-8818-ba96ac47e279.PNG)

We should try and get the install to show the right URL here to the syndesis console.

**Comment by rhuss** [comment]
Yeah, no Idea how the content of the right-side "APPLICATION" (syndesis) and the URL is calulated. I guess of the order of the deployments with routes, but this order seems to be arbitrary.

I searched for a certain label or annotation to apply to influence this info, but to no avail. Maybe @jimmidyson has an idea ?

**Comment by dsimansk** [comment]
Imho renaming the `syndesis-ui` route to just `syndesis` would to the trick. The route list seems to be ordered alphabetically, therefor `syndesis-atlasmap` goes first.

**Comment by rhuss** [comment]
Good idea. Interestingly on 3.5 the order seems to be according to apply order when doing the creation. At least it looks like this on staging.
